backend/date.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
```

Comment:
For clarity and easy reading of the code, I would fix the indentations in this file (and also in the `backend/user.js` file)